### PR TITLE
DPP-1182 Pending Final Approval Notification

### PR DESCRIPTION
**Pull request by lazcool**

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Application.UnitTests/Commands/SubmitCommitment/WhenApprovingAsSecondApprover.cs [hunk 1]
```diff
@@ -83,7 +98,81 @@ public void Setup()
...
+            arg.Email.TemplateId.Should().Be("EmployerTransferPendingFinalApproval");
+            arg.Email.Tokens["first_name"].Should().Be(UserName);
+            arg.Email.Tokens["cohort_reference"].Should().Be(CohortReference);
+            arg.Email.Tokens["sender_name"].Should().Be(senderName);
+            arg.Email.Tokens["provider_name"].Should().Be(providerName);
+            arg.Email.Tokens["employer_hashed_account"].Should().Be(HashedEmployerAccountId);
+
+            var emailBody = PopulateTemplate(template, arg.Email.Tokens);
+            TestContext.WriteLine(emailBody);
+            Assert.AreEqual(expectedEmailBody, emailBody);
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderApprenticeshipsService.Application.UnitTests/Commands/SubmitCommitment/WhenApprovingAsSecondApprover.cs]: As per comment on previous PR, aren't we just testing the output of a private test function?

Comment (lazcool) [reply src/SFA.DAS.ProviderApprenticeshipsService.Application.UnitTests/Commands/SubmitCommitment/WhenApprovingAsSecondApprover.cs]: As per previous reply, we're testing the template is correctly formed and we're storing a copy of the template in our repo.

**Comment by lazcool** [comment]
Chris has verbally approved this PR.

**Comment by satya-sanikommu** [comment]
Review completed, merging this into 'BaseBranch_TransfersNotifications' to test all Transfers notifications features together

**Comment by satya-sanikommu** [comment]
By mistake merged this into Master, as this is a small feature instead of reverting, merged master into 'BaseBranch_TransfersNotifications' to test all Transfers notifications features together

**Pull request merged**